docs/reference/launch-configuration.md

```diff
@@ -36,13 +37,23 @@ The file may be called by any name but should have the following structure:
         }
       ]
     }
-  ]
+  ],
+  "streamingQueryInterval": 4,
+  "runtimeFlags": [
+    {
+      "name": "...",
+      "value": "..."
+    }
+  ]   
 }
 ```
 
 | Field | Required/Optional | Description | 
 | :------------- | :------------- | :------- |
+| `template` | Required | Defines the compute resources your deployment needs (i.e., its ‘topology’). See [deployment templates](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config#templates) for more details. |
```

Comment:
```suggestion
| `template` | Required | Defines the compute resources your deployment needs (its ‘topology’). See [deployment templates](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config#templates) for more details. |
```
(I know this is existing text, but it was bugging me!)

Reply:
Done.